web/src/utils/stringUtils.ts

```diff
@@ -1,8 +1,8 @@
 import normalizeStrings from 'normalize-strings'
 
-export const textTruncator = (text: string, numOfCharsAllowed: number): string => {
+export const textTruncator = (text: string, numOfCharsAllowed: number, replaceNewLineByLineBreak?: boolean): string => {
```

Comment:
🎨 replaceNewLineByLineBreak: boolean = false
So the type is always boolean, not boolean|null

Reply:
Done